### PR TITLE
Update zh_cn.json

### PR DESCRIPTION
**Pull request by Cactusstudent**
sorry be late, just correct the 1.18 version of this mod, the translation is more complete and greater. I cant make sure all of this is correct, but the quality of this transltion is higher than the original version. I need someone to review this. To make sure most of the translation is correct. And I noticed some compat mods dont work, i go to 1.12.2 and then make these. so this part translation I cant make sure is correct.
And i want ChuijkYahus to review this, because I done the TIC3 with him, except me, he is also familiar with TiC3.

---

(edit on 8/12/23) tbh, I want to translate the guide book, but i dont have any time after the summer holiday, so... the Chinese Transaltion of the books may come or not.

### CODE REVIEW

**Comment by wendall911** [comment]
Thank you! I will wait for a review, then will update the mod. I really appreciate the work on this.

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json
Comment (ChuijkYahus) [inline src/main/resources/assets/tinkersurvival/lang/zh_cn.json:65]:
```suggestion
    "material.jelledslime": "凝结黏液",
```
maybe "congealed" is better than "frozen"?
may require a check on recipe or interop with other mods(possibly Tough as Nails) in-game

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json
Comment (ChuijkYahus) [inline src/main/resources/assets/tinkersurvival/lang/zh_cn.json]:
```suggestion
    "GRASS_FIBER_CHANCE.tooltip": "高草丛掉落植物纤维的概率。默认为60%-0.6D。使用小刀的概率再多40%。",
```

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json
Comment (ChuijkYahus) [inline src/main/resources/assets/tinkersurvival/lang/zh_cn.json]:
```suggestion
    "jei.tinkersurvival.description.stick": "破坏树叶时会掉落木棍。使用任何小刀都可以使木棍掉落的概率翻倍。",
```
original text: `Sticks are dropped from leaves. Using any knife doubles the chance of getting a stick.`
fixes the multiplier part

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json
Comment (ChuijkYahus) [inline src/main/resources/assets/tinkersurvival/lang/zh_cn.json:92]:
```suggestion
    "NO_GRIEFING.tooltip": "禁止末影人破坏事物，Notch你在做啥。",
```
This change is only a suggestion
I suppose the first half of sentence is an independent statement from the other half, possibly wrong, no need to change that part if finding it strange to read after editing
The second part is changed to a somewhat question(lit. what are you doing Notch.), maybe it fits the original text better

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json
Comment (ChuijkYahus) [inline src/main/resources/assets/tinkersurvival/lang/zh_cn.json]:
```suggestion
    "FORCE_SAW_FOR_PLANKS.tooltip": "移除原版中木板和木棍的合成配方，且仅能通过锯子制作。默认为true",
```

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json
Comment (ChuijkYahus) [inline src/main/resources/assets/tinkersurvival/lang/zh_cn.json]: `冻结史莱姆` should be changed to match its material name above

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json
Comment (Cactusstudent) [reply src/main/resources/assets/tinkersurvival/lang/zh_cn.json:65]: this material doesnt have any recipe and I even not see it in 1.18 TAN... so i used the translation of 1.12 version.

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json
Comment (Cactusstudent) [reply src/main/resources/assets/tinkersurvival/lang/zh_cn.json]: it's better, thanks

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json
Comment (Cactusstudent) [reply src/main/resources/assets/tinkersurvival/lang/zh_cn.json:92]: I have no idea of this, so I just improve the orginal translation. Maybe we should discuss this

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json
Comment (Cactusstudent) [reply src/main/resources/assets/tinkersurvival/lang/zh_cn.json]: the reason same as jelledslime, not found in 1.18

**Review by ChuijkYahus — APPROVED** [review]

**Comment by Cactusstudent** [comment]
@wendall911 We have done all of these, some keys may not translation 100% sure because some mod not provide this item or material, but we think we did our best. and at last, may I request you to check if this translation have some techical mistake, after you done this, you can merge it. I really appreciate your support of our localization efforts. (and sorry to ping you)

**Pull request merged**
